hardware/Drivetrain.java

```diff
@@ -96,14 +96,14 @@ public void init(HardwareMap hwMap) {
     }
 
     public void teleDrive(double r, double robotAngle, double rightX) {
-        double v1 = r * Math.sin(robotAngle) - rightX;
-        double v2 = r * Math.cos(robotAngle) + rightX;
-        double v3 = r * Math.cos(robotAngle) - rightX;
-        double v4 = r * Math.sin(robotAngle) + rightX;
+        double v1 = r * Math.sin(robotAngle) + rightX;
+        double v2 = r * Math.cos(robotAngle) - rightX;
+        double v3 = r * Math.cos(robotAngle) + rightX;
+        double v4 = r * Math.sin(robotAngle) - rightX;
         setPower(v1,v2,v3,v4);
```

Comment:
I don't know if these changes are valid at all because we never tested teleop. If we can get the rev hub to connect, we will try sometime next week. Getting this correct will be useful for the actual bot.